### ISSUE TITLE
Pie Chart with negative values.

### ISSUE THREAD

**Issue by sensei-Oogway** [knowm/XChart#688]
Hello team,
I've tried to provide some negative values in the Pie chart. Which results in unexpected outcomes. Since pie chart is not applicable for negative values, maybe taking the absolute value before plotting the chart would resolve this issue I think. The following are the steps to reproduce the issue:
1. Create a pie chart with PieChartBuilder.
2. While adding series using addSeries() give in some negative values. Eg;: chart.addSeries("A",-100);

I've also attached the screen shot of the rendered chart with [100,-100,100] as the series.
Thanks and Kindly do the needful.
![image](https://user-images.githubusercontent.com/105499785/168795636-7ff4f095-700e-45a1-bb76-8c3c28ebde96.png)

**Comment by mccartney** [comment]
>  taking the absolute value before plotting the chart would resolve this issue

That's one way to address the problem.
IMHO another and yet cleaner would be to throw an exception for invalid input.

**Comment by timmolter** [comment]
yes, I think there's similar "sanity check" code in other chart types that checks the input and throws an exception when the input is invalid.

**Comment by kalexandru** [comment]
Hi @sensei-Oogway, nice to meet you!
If it is possible I would like to help with this issue, you could assign it to me!
